src/components/SingleName/OldContentWarning.js

```diff
@@ -0,0 +1,5 @@
+export const getOldContentWarning = (type, contentType) => {
```

Comment:
This is not a React functional component, can we remove the caps from the filename? Maybe we can just call it 'warnings.js' and add any other warnings for old content here too